lib/common/schemas.c

```diff
@@ -26,6 +26,7 @@
 #include <crm/common/xml_internal.h>  /* PCMK__XML_LOG_BASE */
 
 #include "crmcommon_private.h"
+#include "pacemaker.h"
```

Comment:
... meaning this won't be needed (libcrmcommon is lower-level than libpacemaker, so it shouldn't include any headers from it)